src/bci_build/package/gcc.py

```diff
@@ -68,6 +68,7 @@ def _is_main_gcc(os_version: OsVersion, gcc_version: _GCC_VERSIONS) -> bool:
             + os_version.lifecycle_data_pkg
         ),
         pretty_name="GNU Compiler Collection",
+        logo_url="https://gcc.gnu.org/img/gccegg-65.png",
```

Comment:
I'm sorry, I don't follow. a logo is not code and an image can't be gplv3+. if you're saying that the containers license is wrong then I agree, but how does adding the logo change the wrong license declaration of the container?

Reply:
Does the image end in the final image as a binary inside it? If the image is used by a label as a URL there's no real issue here I think, even if the image would be used by the registry, it points to an external source.

I'm not sure where you can find the right license for this logo, but a quick search didn't result in anything, it might be under the Free Art License as many other images from FSF.

Reply:
No, of course not. the only thing that ends up in the final image is the "https:/.." url. hence I'm really confused on what the issue is. a link to an image has never changed the software license for me.